public/app/percona/add-instance/components/AddRemoteInstance/AddRemoteInstance.service.tsx

```diff
@@ -175,6 +175,9 @@ export const toPayload = (values: any, discoverName?: string, type?: InstanceAva
   if (type === Databases.mongodb && values.tls) {
     data.authentication_mechanism = 'MONGODB-X509';
   }
+  if (type === Databases.mongodb && values.disable_collectors) {
```

Comment:
since there's a common condition between this block and the one above (`type === Databases.mongodb`), can we merge it together?

Reply:
I think yes) I will move it)